EpsilonDriverControls/Inc/DriverControls.h

```diff
@@ -7,15 +7,34 @@
 
 #define HEARTBEAT_CAN_FREQ 1000
 #define HEARTBEAT_STDID 0x700U
+#define HEARTBEAT_DLC 1
 
 #define LIGHTS_CAN_FREQ 100
 #define LIGHTS_STDID 0x701U
+#define LIGHTS_DLC 1
 
 #define MUSIC_CAN_FREQ 200
 #define MUSIC_STDID 0x702U
+#define MUSIC_DLC 1
 
 #define DRIVER_CAN_FREQ 50
 #define DRIVER_STDID 0x703U
+#define DRIVER_DLC 4
+
+#define DRIVE_COMMANDS_FREQ 100
+#define MOTOR_DRIVE_STDID 0x501U
+#define MOTOR_DRIVE_DLC 8
+#define MOTOR_POWER_STDID 0x502U
+#define MOTOR_POWER_DLC 8
+#define MOTOR_RESET_STDID 0x503U
+
+#define ADC_POLL_TIMEOUT 1000
+
+#define MAX_FORWARD_RPM 1000 //Was 1298 on old system
```

Comment:
`//Was 1298 on old system` I think this comment does not need to make it over. What first gen CCS did is no longer relevant =P